.config/dotnet-tools.json

```diff
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "fake-cli": {
-      "version": "5.22.0",
+      "version": "5.23.0-alpha002",
```

Comment:
@Krzysztof-Cieslak this prerelease fixed FAKE on newer .NET runtimes for me.

Reply:
I hit this issue on macOS/M1 when trying to build fsac a couple of months ago too